### DISCUSSION TITLE
Sentence Similarity Task Sample Example Error

### DISCUSSION THREAD

**Discussion by mayurjay** [autogluon/autogluon#1647] [Q&A]
I tried running the sample example of sentence similarity task available at https://auto.gluon.ai/stable/tutorials/text_prediction/beginner.html

The following code raises an error:
```
predictor_sts = TextPredictor(label='score', path='./ag_sts')
predictor_sts.fit(sts_train_data, time_limit=60)
```
The error message is as follows:
`ValueError: The `target` has to be an integer tensor.`

Please let me know how to fix this problem.

**Comment by sxjscience** [comment]
Would you post the complete error message? Also, you may attach the values from the following command:

```python
# Replace this code with the output of the following:
from autogluon.core.utils import show_versions
show_versions()
```
